nexus/reconfigurator/planning/src/system.rs

```diff
@@ -121,6 +122,10 @@ impl SystemDescription {
 
         // Policy defaults
         let target_nexus_zone_count = NEXUS_REDUNDANCY;
+        // TODO-cleanup This is wrong, but we don't currently set up any CRDB
+        // nodes in our fake system, so this prevents downstream test issues
+        // with the planner thinking our system is out of date from the gate.
+        let target_cockroachdb_zone_count = 0;
```

Comment:
I was also wondering about this when looking at our tests the other day.  Should we actually deploy CRDB zones and everything else as part of our `ExampleSystem`? It feels like we should. Not in this PR though.

Reply:
Agreed, and initially I set this to `COCKROACHDB_REDUNDANCY`. That broke basically all the nontrivial tests, though, so I took the easy way out and punted on this.

Reply:
Makes sense.

Reply:
I opened an issue for this: https://github.com/oxidecomputer/omicron/issues/5808